spacy_experimental/biaffine_parser/arc_predicter.pyx

```diff
@@ -147,27 +160,33 @@ class ArcPredicter(TrainablePipe):
 
     def predict(self, docs: Iterable[Doc]):
         docs = list(docs)
-        lens = sents2lens(docs, ops=self.model.ops)
-        scores = self.model.predict((docs, lens))
 
-        lens = to_numpy(lens)
+        if self.senter:
+            lengths = split_lazily(docs, ops=self.model.ops, max_length=self.max_length, senter=self.senter, is_train=False)
+        else:
+            lengths = sents2lens(docs, ops=self.model.ops)
```

Comment:
Do we make sure somewhere that in this case we do have `doc.sents`?

Reply:
spaCy raises a nice error message when you try to use `sents` when it is not present:

https://github.com/explosion/spaCy/blob/6aa6b86d496c8d9271f42c077a79f9bfb88687ac/spacy/tokens/doc.pyx#L891